packages/terra-list/docs/MultiSelect.md

```diff
@@ -0,0 +1,16 @@
+# Multi Select List
+
+The Terra List is a structural component to arrange content within list/listitems with a configurable number of selectable items.
```

Comment:
Update to say Terra Multi Select List

Reply:
https://github.com/cerner/terra-core/pull/853/commits/6c35ba1ab5c8608c7f9c3fdc74081d6512d96d8c